nexus/external-api/src/lib.rs

```diff
@@ -1386,22 +1494,59 @@ pub trait NexusExternalApi {
 
     /// Remove instance from a multicast group.
     ///
+    /// The group can be specified by name or UUID.
+    #[endpoint {
+        method = DELETE,
+        path = "/v1/multicast-groups/{multicast_group}/members/{instance}",
+        tags = ["experimental"],
+        operation_id = "multicast_group_member_remove",
+        versions = ..VERSION_MULTICAST_IMPLICIT_LIFECYCLE_UPDATES,
+    }]
+    async fn v2025120300_multicast_group_member_remove(
+        rqctx: RequestContext<Self::Context>,
+        path_params: Path<v2025120300::MulticastGroupMemberPath>,
+        query_params: Query<params::OptionalProjectSelector>,
+    ) -> Result<HttpResponseDeleted, HttpError>;
+
+    /// Remove instance from a multicast group.
+    ///
+    /// The group can be specified by name, UUID, or multicast IP address.
+    /// All three are fleet-wide unique identifiers.
+    ///
     /// Functionally equivalent to removing the group from the instance's
     /// `multicast_groups` field. Both approaches modify the same underlying
     /// membership and trigger reconciliation.
     ///
+    /// Authorization: requires Modify on the instance (checked first) and Read
+    /// on the multicast group.
+    ///
     /// Specify instance by name (requires `?project=<name>`) or UUID.
     #[endpoint {
         method = DELETE,
         path = "/v1/multicast-groups/{multicast_group}/members/{instance}",
         tags = ["experimental"],
+        versions = VERSION_MULTICAST_IMPLICIT_LIFECYCLE_UPDATES..,
     }]
     async fn multicast_group_member_remove(
```

Comment:
Same question as above, do we need this when we have the instance leave endpoint?